src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+
+  padding: 32px 16px 16px;
+
+  border: 1px solid $c-shadow;
+  border-radius: 5px;
+
+  // &:not(:nth-child(4n)) {
+  //   margin-right: 48px;
+  // }
+  // &:nth-child(4n) {
+  //   margin-right: calc((100% - 200px * 4 - 50px * 3 )/ 2);
+  // }
+
+  // &:nth-last-child(n + 5) {
+  //   margin-bottom: 46px;
+  // }
+
+  // &:nth-last-child(4n - 4) {
+  //   margin-left: calc((100% - 200px * 4 - 50px * 3 )/ 2);
+  // }
+
+  &__product-image {
+    display: block;
+
+    width: 160px;
+    height: 134px;
+
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    @extend %text-normal;
+    font-weight: 500;
+    color: $c-main;
+
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    @extend %text-small;
+    font-weight: 400;
+    color: $c-secondary;
+
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    @extend %flex-between;
+
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    @extend %text-small;
+    color: $c-main;
+    align-self: flex-end;
+  }
+
+  &__price {
+    @extend %flex-between;
+
+    margin-bottom: 16px;
+
+    &-text {
+      @extend %text-normal;
+      color: $c-secondary;
+    }
+
+    &-value {
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+      color: $c-main;
+    }
```

Comment:
Wrong `&` usage, use it to avoid repeating block names in class, but not just for combining words, it makes code less readable